apis/v1alpha1/gateway_types.go

```diff
@@ -466,54 +468,45 @@ type RouteNamespaces struct {
 
 // GatewayAddress describes an address that can be bound to a Gateway.
 type GatewayAddress struct {
-	// Type of the Address. This is either "IPAddress" or "NamedAddress".
+	// Type of the address.
 	//
 	// Support: Extended
 	//
 	// +kubebuilder:default=IPAddress
 	Type AddressType `json:"type,omitempty"`
 
-	// Value. Examples: "1.2.3.4", "128::1", "my-ip-address". Validity of the
-	// values will depend on `Type` and support by the controller.
+	// Value of the address.
+	// The validity of the  values will depend on the type and support by the controller.
+	//
+	// Examples: "1.2.3.4", "128::1", "my-ip-address".
```

Comment:
Nit: The indentation here feels a bit off to me. I generally try to wrap lines at 80 chars - instead this seems to add a new line for each sentence.

Reply:
Pretty sure I addressed all the cases of this.